### DISCUSSION TITLE
Is Editing AcroForms possible?

### DISCUSSION THREAD

**Discussion by Richard-Woessner** [UglyToad/PdfPig#764] [Q&A] [answered]
Sorry if this is available somewhere and I can't find it but I'm looking to edit an AcroFormField's value. I'm successfully returning it like so;

```
                bool hasForm = document.TryGetForm(out AcroForm form);

                var fields = form.GetFields();

                AcroFieldBase dda = fields
                    .Where(x => x.Information.PartialName == "DDA")
                    .FirstOrDefault();
``` 

Can someone point me to the right direction?

**Comment by EliotJones** [comment, marked as answer]
Hi unfortunately this isn't possible with this library at the moment. Editing is still not full implemented yet. I'm not sure which the best libraries are that support editing for PDFs to be honest sorry.

**Reply by Richard-Woessner** [reply]
All good, I understand
